vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -209,6 +209,7 @@ type Info struct {
 	Plugins            PluginsInfo
 	MemoryLimit        bool
 	SwapLimit          bool
+	KernelMemory       bool
```

Comment:
can you drop this if it isn't related to userns=host? otherwise we'll introduce it now in 1.10.3